### DISCUSSION TITLE
How do I change the path for SMPlayer Portable?

### DISCUSSION THREAD

**Discussion by Galathorne** [mach64/trakt-scrobbler#241] [Installation help] [answered]
I am currently using Windows 10 and have completed all the necessary setup steps. However, I am having trouble getting my custom SMPlayer folder to work with Trakt Scrobbler.

When I try to start or restart the service, I get an error message that says
`...Failed to autoload value from 'C:\\Users\\gmich\\.smplayer\smplayer.ini': File not found`.

My SMPlayer folder, which includes the mpv program, is located at `D:\! Portable Apps\! SMPlayer Portable\` and I am not sure how to set this folder for use with Trakt Scrobbler.

As a temporary solution I am trying, I have been starting SMPlayer using a batch file with the following commands:

> @echo off
copy smplayer.ini C:\Users\gmich\smplayer\smplayer.ini
smplayer.exe

This ensures that the latest ini file is always used and that the error does not show, but I am unsure if this (not being my working SMPlayer folder) is causing issues with the scrobbler. I have not yet seen any of the videos I have watched being added to my Trakt history, even though all of the watch folders have been set up correctly. I am not sure how the scobbler captures the watched items, but this workaround I am trying seems to indicate that it is not using the `History` section of the smplayer.ini file, else it would have probably sync'd correctly.

How should I proceed getting my correct working SMPlayer folder set up for Trakt Scrobbler?

**Comment by iamkroot** [comment, marked as answer]
Simply put, you don't need to move or copy any smplayer files. Instead, tell this app about the actual values it wants to.
See https://github.com/iamkroot/trakt-scrobbler/wiki/Players-Setup#wrappers

Specifically, run `trakts config set players.smplayer@mpv.ipc_path \\.\pipe\mpvsocket` after following https://github.com/iamkroot/trakt-scrobbler/wiki/Players-Setup#smplayer
